openshift_3_x/bundle.yaml

```diff
@@ -6,12 +6,20 @@ metadata:
 allowPrivilegedContainer: false
```

Comment:
that folder should have been removed. I will open a ticket to do it next release.